src/packages/shared/src/components/select/component.js

```diff
@@ -18,15 +19,19 @@ const Select = ({
   ...rest
 }) => {
   const Component = creatable ? ReactSelectCreatable : ReactSelect;
+  const [isDisabled, setIsDisabled] = useState(disabled || loading);
 
   return (
     <Component
       inputId={id}
       aria-label={ariaLabel}
       options={options}
       value={value}
-      onChange={onChange}
-      isDisabled={disabled}
+      onChange={value => {
```

Comment:
This change is breaking all the selects of the widget-editor. For example, the X and Y axis columns can't be changed anymore once set.

It even affects the filters themselves: the user is not able to change a filter's operation anymore.